flocker/node/_loop.py

```diff
@@ -231,6 +233,21 @@ class ConvergenceLoopOutputs(Names):
     CONVERGE = NamedConstant()
 
 
+_FIELD_CONNECTION = Field(
+    u"connection",
+    lambda client: repr(client),
+    "The AMP connection to control service")
+
+LOG_SEND_TO_CONTROL_SERVICE = ActionType(
+    u"flocker:agent:send_to_control_service",
+    # XXX fields would just be the address of the control service, we can
+    # rely on the higher-level logged action of the convergence loop to
+    # log the local state (in FLOC-1379).
+    # XXX what's the success field here?
+    [_FIELD_CONNECTION], [],
```

Comment:
Could be address of control service? Or not if that's easier, for now, since in short term there's only one control service address anyway. Oh, another useful thing would be the unique identifier for the convergence loop FSM (`fsm.identifier()`). But you can feel free to omit this in this branch and just have comment referring to followup.

So, some stuff in there would be useful, but not necessary to complete this branch. So just omit it.